src/docs/user/references/GenericInformation.adoc

```diff
@@ -207,6 +207,14 @@ The following table describes all available generic information:
 |declare a string to be used as options separator in <<../user/ProActiveUserGuide.adoc#_dockerfile,Dockerfile script engine>>.
 |job-level, task-level
 |!SPLIT!
+|<<_job_planner,calendar.name>>
+|generic information automatically added by the Job Planner. It contains the name of the calendar based on which the job is planned.
```

Comment:
Shouldn't we user plural form of job ? ... the calendar based on which the jobs are planned.

Reply:
It contains the name of the calendar based on which the job is planned. => Sentence difficult to read

Reply:
The GI are per job, so I prefer keeping the singular mode to be more precise. Thanks @marounkoussaifi